lib/openhab/dsl/rules/builder.rb

```diff
@@ -964,33 +964,68 @@ def every(value, at: nil, attach: nil)
         #
         # Run this rule when the script is loaded.
         #
-        # Execute the rule on OpenHAB start up and whenever the script is
-        # reloaded. This is useful to perform initialization routines,
-        # especially when combined with other triggers.
+        # Execute the rule whenever the script is first loaded, including on OpenHAB start up,
```

Comment:
s/OpenHAB/openHAB/ (yes, I relatively recently learned that's the proper capitalization for the proper noun. Obviously we can't use that for the OpenHAB module, but we should refer to it that way in any documentation.

Reply:
We'll do a mass replacement in a separate PR for this, I think.